css/pages/navbar.css

```diff
@@ -90,7 +94,7 @@ nav li:hover a{
 
```

Comment:
Can we please get rid of these extra spaces (I know, they're not a part of your PR, but still)

Reply:
@swarajpure  updated.